src/lib/schema.mjs

```diff
@@ -70,6 +71,16 @@ export async function gatherRelativeFSLFilePaths(dir) {
   return files;
 }
 
+export async function deleteUnusedSchemaFiles(dir, filesToDelete) {
+  const fs = container.resolve("fs")
+  const promises = []
+  for (const fileName of filesToDelete) {
+    promises.push(fs.unlink(path.join(dir, fileName)))
+  }
+
+  return Promise.all(promises)
+}
```

Comment:
parallelized re-implementation of one of the schema helpers (you'll see this pattern a few times).